build.gradle

```diff
@@ -5,15 +5,15 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.3'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31"
-        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:2.3.5"
+        classpath 'com.android.tools.build:gradle:8.0.2'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.21"
+        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:2.5.3"
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
-task clean(type: Delete) {
+tasks.register('clean', Delete) {
```

Comment:
<!-- コード理解・解説のためのメモ書き -->
![memo-badge](https://img.shields.io/badge/review-memo-lightgrey) 

以下の Warning が出ていたため `tasks.register` に変更しています。

```text
Consider using 'tasks.register' to avoid unnecessary configuration
```